backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/textureview/GLTextureView.java

```diff
@@ -0,0 +1,1705 @@
+package com.badlogic.gdx.backends.android.textureview;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.graphics.SurfaceTexture;
+import android.opengl.GLDebugHelper;
+import android.opengl.GLSurfaceView;
+import android.os.SystemClock;
+import android.util.Log;
+import android.view.KeyCharacterMap;
+import android.view.KeyEvent;
+import android.view.TextureView;
+import android.view.View;
+import android.view.inputmethod.BaseInputConnection;
+import android.view.inputmethod.EditorInfo;
+import android.view.inputmethod.InputConnection;
+
+import com.badlogic.gdx.backends.android.surfaceview.GdxEglConfigChooser;
+import com.badlogic.gdx.backends.android.surfaceview.ResolutionStrategy;
+
+import java.io.Writer;
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.microedition.khronos.egl.EGL10;
+import javax.microedition.khronos.egl.EGL11;
+import javax.microedition.khronos.egl.EGLConfig;
+import javax.microedition.khronos.egl.EGLContext;
+import javax.microedition.khronos.egl.EGLDisplay;
+import javax.microedition.khronos.egl.EGLSurface;
+import javax.microedition.khronos.opengles.GL;
+import javax.microedition.khronos.opengles.GL10;
+
+/*
+ * Copyright (C) 2018 Wasabeef
```

Comment:
This seems to be a 3rd party library. (Not necessarily a problem, but just to make it clear, since the PR text does not.)

Reply:
Yes, I took this class from other project and modified a little. Wasn't sure about copyright and how to deal with it, so left it as it was. Should it be replaced or modified somehow?